src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml.cs

```diff
@@ -102,30 +102,57 @@ internal void DependencyRegen(WorkspaceModel ws)
         {
             // Clear the dependency table.
             table.Columns.Clear();
-            foreach (var package in ws.NodeLibraryDependencies)
+            foreach (PackageDependencyInfo package in ws.NodeLibraryDependencies)
```

Comment:
not all objects in `NodeLibraryDependencies` will be `PackageDependencyInfo` - I think it would be good to check the type - ie - if dependency is `PackageDependencyInfo` - NodeLibraryDependencies should contain a list of `INodeLibraryDependencies`

Reply:
Agree, no need to be specific here

Reply:
Changed this to check for the type PackageDependencyInfo. Should the other objects be displayed in the dependency table or just those which are of PackageDependencyInfo type?

Reply:
only show the PackageDep objects for now, the others don't exist yet 😉